src/server_api.ts

```diff
@@ -1,9 +1,57 @@
-import { Asset, AssetType } from "stellar-base";
+import { Asset } from "stellar-base";
 import { Omit } from "utility-types";
 import { Horizon } from "./horizon_api";
 
+// more types
+import { AccountRecordSigners as AccountRecordSignersType } from "./types/account";
+import { AssetRecord as AssetRecordType } from "./types/assets";
+import {
```

Comment:
When we're importing this much stuff, maybe: `import * as effects from "./types/effects";` might be simpler?

Reply:
I let Code fill the import for me from Effects, but it makes sense since there is a lot imported from that file. thanks for the suggestion.